index.html

```diff
@@ -1,10 +1,192 @@
+        <div class="header-squiggle"></div>
+    </header>
+
+    <main>
+        <div class="main-content-wrapper">
+            <section class="hello" id="first-lines">
+                <section class="info-banner">
+                    <h5>Rekupe is a digital agency in Los Angeles.</h5>
+                    <h5>We use strategic UX, SEO optimization and beautiful<br>
+                        visual designs to help your business grow.</h5>
+                    <h5>We see every project as a fresh opportunity,<br>
+                        pushing ourselves to create new digital experiences.</h5>
+
+                </section>
+                <p class="contact-link" id="contact-black">Have a question? <a href="#contact-form" class="nodeco">Click here</a> to contact us!</p>
+            </section>
+
+            <section class="projects-section" id="casestudies">
+
+
+                <article class="project " id="rolling-stone">
+                    <span class="img-wrapper"><img src="assets/work-image-01.jpeg" alt="Rolling Stone Image" class="project-image"></span>
+
+                    <div class="project-content">
+                        <h3>Rolling Stone</h3>
+                        <span class="squiggle-wrapper"><img src="assets/squiggle_line.svg"></span>
+                        <p>You know Rolling Stone. It’s the definitive <br>
+                            voice in music, politics, and culture. It’s a <br>
+                            global icon.</p>
+                        <span class="projectview">View Project ― </span>
+                    </div>
+                </article>
+
+                <article class="project order2" id="variety">
+                    <span class="img-wrapper"><img src="assets/work-image-02.jpeg" alt="Rolling Stone Image" class="project-image"></span>
+                    <div class="project-content">
+                        <h3>Variety</h3>
+                        <p>Variety is an industry leader for breaking entertainment news, reviews, and box office results with an archive of content dating back to 1905.</p>
+                        <span class="projectview">View Project ― </span>
+                    </div>
+                </article>
+
+                <article class="project" id="worth">
+                    <span class="img-wrapper"><img src="assets/work-image-03.jpeg" alt="Variety Image" class="project-image"></span>
+                    <div class="project-content">
+                        <h3>Worth</h3>
+                        <p>A global media platform connecting a community that embraces worth beyond wealth.</p>
+                        <span class="projectview">View Project ― </span>
+                    </div>
+
+                </article>
+            </section>
+
+
+            <section class="services-wrapper" id="services">
+
+                <article class="serviceblock" id="web-design">
+                    <div class="img-for-card"><img src="assets/icon_web-design.svg">
```

Comment:
Omdat de icoontjes altijd dezelfde grootte blijven (ook op mobiel) hoef je ze niet responsive te maken en daarom ook niet in een div- of span te wikkelen. Deze mag hier daarom weg :)